cloud/app/server/server.js

```diff
@@ -1240,6 +1294,10 @@ class _robotAgent extends Capability {
       res.json(this.getLatestRunningVersions(req.session.user._id));
     });
 
+    this.router.get('/capabilitiesInUse', requireLogin, async (req, res) => {
```

Comment:
I assume this is meant for use on the /admin page. If so I think we should put it behind `requireAdmin`.